src/utils/fast-apply.js

```diff
@@ -0,0 +1,19 @@
+// OPTIMIZATION: http://jsperf.com/call-apply-optimization
```

Comment:
Nevermind. Seems like it's used only by client code. Let's move it to `src/client/utils/ then